test/helpers/random.ts

```diff
@@ -0,0 +1,11 @@
+import { randomUUID, randomBytes } from "crypto"
+
+export const randomEmail = () =>
+  (randomBytes(20).toString("hex") + "@galoy.io") as EmailAddress
+
+export const randomPassword = () => randomBytes(20).toString("hex") as IdentityPassword
+
+export const randomPhone = () =>
```

Comment:
This was refactored and separated out as a functional grouping because original grouping by concept in `index.ts` introduced integration-level dependencies that broke the unit tests on import